README.new-mail

```diff
@@ -0,0 +1,48 @@
+new-mail-command Patch
+======================
+
+    Configure a custom command to be executed when a new email is received.
+
+Patch
+-----
+
+    To check if Mutt supports "new-mail-command", look for
+    "new-mail" in the mutt version.
```

Comment:
@pierremoreau Thanks for your changes.

> Mutt seems to be supporting it, even if I do not get new-mail (or similar) in the output of mutt -v

The last release was a bit rushed and nothing quite agrees.
The next release (soon) will be a little more polished :-)

> Nor could I find any option to enable/disable it at compile time.

Generally, that's only been done for the larger, more invasive features.
new-mail is a small feature that's unlikely to cause problems.